React/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "test-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "devextreme": "^20.2.5",
+    "devextreme-react": "^20.2.5",
+    "react": "^17.0.1",
```

Comment:
It is necessary to downgrade to 16, for now, bcs of the bug in our React integration. Also, a peer-dep error may occur